Formula/erlang.rb

```diff
@@ -8,6 +8,7 @@ class Erlang < Formula
 
   bottle do
     cellar :any
+    rebuild 1
```

Comment:
This is just a rebuild with a flag change needed for High Sierra. There are no changes. It's not a new version. 

The new compiler flag prevents linking against BoringSSL on 10.13. It should have no effect on any other OS and the current behaviour on 10.13 is a bug.

I think bumping versions will confuse people hence the rebuild.